cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/system/CDAPLogAppender.java

```diff
@@ -66,7 +66,15 @@ public class CDAPLogAppender extends AppenderBase<ILoggingEvent> implements Flus
   private int fileRetentionDurationDays;
   private int fileCleanupBatchSize;
 
+  private final boolean disableLogCleanerForTest;
+
+  @VisibleForTesting
+  CDAPLogAppender(boolean disableLogCleanerForTest) {
+    this.disableLogCleanerForTest = disableLogCleanerForTest;
```

Comment:
Call   `setName(getClass().getName())`, and the the other constructor can call this one with false param

Reply:
Done, thanks!